tests/common.py

```diff
@@ -13,6 +17,17 @@
 import itertools
 
 
+def get_default_ff(default_ff='ff/params/smirnoff_1_1_0_ccc.py'):
```

Comment:
How would this change if we upgraded to say 1.3.0 later on? Should we rename this to be something more specific like 
`get_110_ccc_ff()`?

Reply:
>How would this change if we upgraded to say 1.3.0 later on?

Same way we would change the other uses in the project-- this still pops up if we find and replace the literal `ff/params/smirnoff_1_1_0_ccc.py` in all project files. Later if we want to have this be more automatic, we could maybe set `DEFAULT_FF = "ff/params/smirnoff_1_1_0_ccc.py"` to be some project-level constant (which could replace the ~ two dozen duplicates.

>Should we rename this to be something more specific like get_110_ccc_ff()?

Applied this name change in https://github.com/proteneer/timemachine/pull/490/commits/c1243d724475754c8ededcb0b5aac02a25acb6db